apps/server/src/modules/tldraw/controller/api-test/tldraw.controller.api.spec.ts

```diff
@@ -13,19 +15,29 @@ const baseRouteName = '/tldraw-document';
 describe('tldraw controller (api)', () => {
 	let app: INestApplication;
 	let em: EntityManager;
-	let testApiClient: TestApiClient;
+	let testXApiKeyClient: TestXApiKeyClient;
+	const API_KEY = '7ccd4e11-c6f6-48b0-81eb-cccf7922e7a4';
 
 	beforeAll(async () => {
 		const module: TestingModule = await Test.createTestingModule({
 			imports: [ServerTestModule],
 			controllers: [TldrawController],
 			providers: [Logger, TldrawService, TldrawRepo],
-		}).compile();
+		})
+			.overrideGuard(AuthGuard('api-key'))
+			.useValue({
+				canActivate(context: ExecutionContext) {
+					const req: Request = context.switchToHttp().getRequest();
+					req.headers['X-API-KEY'] = API_KEY;
+					return true;
+				},
+			})
```

Comment:
After working with the testXAPIKeyClient it should be possible to remove this code lines.

Reply:
Unfortunatelly it is not, even after ticket bc-4100 improvements. Most of test files that uses testXAPIKeyClient nevetheless also has those lines of code. I've contacted team Solaris about this and received info that there is ticket created to deal with this issue and for imporoving testXAPIKeyClient:
https://ticketsystem.dbildungscloud.de/browse/BC-5903

Reply:
I put it to the left over list.